### PR TITLE
added an optional meta object param to all sign transactions

### PR DESCRIPTION
**Pull request by IAmMorrow**
#149 

This meta object can be used to pass wallet specific data

### CODE REVIEW

**Comment by adrienlacombe-ledger** [comment]
https://github.com/LedgerHQ/wallet-api/issues/149

File: packages/server/src/types.ts [hunk 1]
```diff
@@ -62,16 +62,19 @@ export interface WalletHandlers {
   "message.sign": (params: {
     account: Account;
     message: Buffer;
+    meta?: unknown;
```
Comment (Justkant) [inline packages/server/src/types.ts:65]:
```suggestion
    meta?: Record<string, unknown>;
```

File: packages/server/src/types.ts [hunk 2]
```diff
@@ -62,16 +62,19 @@ export interface WalletHandlers {
...
   }) => Promisable<Buffer>;
   "transaction.sign": (params: {
     account: Account;
     transaction: Transaction;
     options?: TransactionSign["params"]["options"];
+    meta?: unknown;
```
Comment (Justkant) [inline packages/server/src/types.ts:71]:
```suggestion
    meta?: Record<string, unknown>;
```

File: packages/server/src/types.ts [hunk 3]
```diff
@@ -62,16 +62,19 @@ export interface WalletHandlers {
...
   }) => Promisable<Buffer>;
   "transaction.signAndBroadcast": (params: {
     account: Account;
     transaction: Transaction;
     options?: TransactionSignAndBroadcast["params"]["options"];
+    meta?: unknown;
```
Comment (Justkant) [inline packages/server/src/types.ts:77]:
```suggestion
    meta?: Record<string, unknown>;
```

**Review by Justkant — CHANGES_REQUESTED** [review]
Only one small change

**Review by Justkant — APPROVED** [review]

**Pull request merged**